components/About/AboutPage.js

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import NavigationBar from '../Common/NavigationBar';
+import { ABOUT_STRING } from './constants';
+import { HOME_STRINGS } from '../constants';
+import './about.scss';
+
+const {
+    INTRO_HEADER_TEXT,
+    NINTY,
+    TEST1,
+    TEST2,
+    CLIENT,
+    INTRO_TEXT1,
+    INTRO_TEXT2,
+} = ABOUT_STRING;
+
+const {
+    READ_MORE,
+    LIFE,
+    CONSULT_TEXT1,
+    CONSULT_TEXT2,
+} = HOME_STRINGS;
+
+const AboutPage = () => {
+    return (
+        <div>
+            <div className="header">
+                <NavigationBar />
+                <div className="header-cover">
+                    <span className="header-cover-text">
+                        <p className="text1">About </p>
+                        {' '}
+                        <p className="text2">{LIFE}</p>
+                    </span>
+                    <p className="desc">Home  /  About Lifeaid</p>
+                </div>
+            </div>
+            <div className="intro app-content">
+                <hr />
+                <p className="intro-header">
+                    {INTRO_TEXT1}
+                    <br />
+                    {' '}
+                    {INTRO_TEXT2}
+                </p>
+                <span>
+                    <p>
+                        {INTRO_HEADER_TEXT}
+                        <br />
+                        <br />
+                        <br />
```

Comment:
why are you abusing the `br` tag element, why don't you just use css to create the needed `margin`

Reply:
The css margin wasn't giving me the exact precise space I needed